<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-GGLBD97Q" failures="2" tests="6" name="Test" time="112.699" errors="0" timestamp="2022-01-12T08:39:42 IST">
  <testcase classname="com.application.test.ApplicationTest" name="verifyApplicationTitle" time="3.883"/>
  <testcase classname="com.application.test.ApplicationTest" name="verifyApplicationLogin" time="8.878"/>
  <testcase classname="com.application.test.ApplicationTest" name="verifyApplicationLogin" time="37.019">
    <failure type="java.lang.AssertionError" message="Application Login is not Successful">
      <![CDATA[java.lang.AssertionError: Application Login is not Successful
at com.application.pages.LoginPage.verifySuccessfulLogin(LoginPage.java:91)
at com.application.test.ApplicationTest.verifyApplicationLogin(ApplicationTest.java:31)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- verifyApplicationLogin -->
  <testcase classname="com.application.test.ApplicationTest" name="verifyApplicationLogin" time="37.191">
    <failure type="java.lang.AssertionError" message="Application Login is not Successful">
      <![CDATA[java.lang.AssertionError: Application Login is not Successful
at com.application.pages.LoginPage.verifySuccessfulLogin(LoginPage.java:91)
at com.application.test.ApplicationTest.verifyApplicationLogin(ApplicationTest.java:31)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- verifyApplicationLogin -->
  <testcase classname="com.application.test.ApplicationTest" name="verifyForgotPasswordPage" time="5.136"/>
  <testcase classname="com.application.test.ApplicationTest" name="verifyApplicationLogout" time="9.784"/>
</testsuite> <!-- Test -->
